CODE_OF_CONDUCT.md

```diff
@@ -6,7 +6,7 @@ In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to make participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
 size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
+level of experience, education, socioeconomic status, nationality, personal
```

Comment:
I guess this is a bit like changing the wording in a LICENSE file. The code of conduct is versioned and verbatim copy of what's been put out by the contributor-covenant, so I think this should be kept as-is. They repeat the same spellign in newer versions, it seems.